packages/lightning-language-server/package.yaml

```diff
@@ -0,0 +1,17 @@
+
+name: lwc-language-server
+description: Language Server for Lightning Web Components.
+homepage: https://github.com/forcedotcom/lightning-language-server/
+licenses:
+  - BSD-3-Clause
+languages:
+  - HTML
+  - JavaScript
+categories:
+  - LSP
+source:
+  id: pkg:npm/@salesforce/lwc-language-server@v4.7.3
+schemas:
+  lsp: vscode:https://raw.githubusercontent.com/forcedotcom/lightning-language-server/main/packages/lwc-language-server/package.json
```

Comment:
This package.json doesn't seem to include any relevant information ([`"contributes.configuration"`](https://code.visualstudio.com/api/references/contribution-points#contributes.configuration)), so we can exclude it.
```suggestion
```